### PR TITLE
Update to Fabric 1.21

### PR DESCRIPTION
**Pull request by Soumeh**
- Fabric (i think) supports Quilt too, so the mod can be used on both loaders.
- Will (probably) support [Sinytra Connector](https://modrinth.com/mod/connector/versions) whenever it updates.
- Maybe works with other Minecraft versions? I removed any mc version limitations but i'm not sure how the fabric loader will handle the mod.

### CODE REVIEW

**Comment by SwanX1** [comment]
Please make the PR merge into the `1.19-quilt` branch, otherwise the code diff is unreadable...

**Comment by Soumeh** [comment]
ah my bad, i changed it now

File: .editorconfig
Comment (SwanX1) [inline .editorconfig:1]: please keep this file

File: gradle.properties [hunk 1]
```diff
@@ -1,6 +1,10 @@
-org.gradle.jvmargs = -Xmx1G
-org.gradle.parallel = true
-
-version=1.0.2
+mod_id=elytrabounce
 maven_group=org.infernalstudios
-archives_base_name=elytrabounce
\ No newline at end of file
+mod_version=1.1.0
```
Comment (SwanX1) [inline gradle.properties]: no need to increment the version, if you're just porting

File: src/main/resources/assets/elytrabounce/META-INF/mods.toml
Comment (SwanX1) [inline src/main/resources/assets/elytrabounce/META-INF/mods.toml:1]: ???

this is a fabric mod

**Review by SwanX1 — CHANGES_REQUESTED** [review]
few minor changes

**Review by SwanX1 — APPROVED** [review]

**Comment by SwanX1** [comment]
Merged into new branch `fabric-1.21`

**Pull request closed without merging**